viewer/v2client/config/api_config.json.in

```diff
@@ -1,5 +1,6 @@
 {
   "path": "http://kblocalhost.kb.se:5000",
   "auth": "https://login.libris.kb.se/oauth/verify",
-  "id": "http://localhost:5000"
+  "id": "http://localhost:5000",
+  "piwik_id": "00"
```

Comment:
We should update the devops scripts to add `piwik_id` to the config file.

Reply:
👍